bobtemplates/hooks.py

```diff
@@ -31,3 +31,19 @@ def post_package_name(configurator, question, answer):
 def pre_skip_configure_deployment(configurator, question):
     if not configurator.variables['package.configure_deployment']:
         raise(SkipQuestion)
+
+
+def get_plone_classifier_version(configurator, question, answer):
+    try:
+        if answer.split('.')[2]:
+            classifier_version = '.'.join(answer.split('.')[:2])
+    except IndexError:
+        try:
+            if answer.split('.')[1]:
+                classifier_version = answer
+        except IndexError:
+            raise ValidationError(
+                'The plone version needs to have at least 2 digits e.g.: 4.3')
+
+    configurator.variables['package.classifier_version'] = classifier_version
+    return answer
```

Comment:
- Bitte die gesplittete answer in einer Variable speichern.
- Den `IndexError` kannst du vermeiden:

```bash
>>> '.'.join(["4", "3", "8"][:2])
'4.3'
>>> '.'.join(["4", "3"][:2])
'4.3'
>>> '.'.join(["4"][:2])
'4'
```

-Für den `ValidationError` kannst du die Länge vom Split-Resultat anschauen, wenns kleiner oder gleich null ist wird ein `ValidationError` geraised.